src/dashboard/src/requirements/dev.txt

```diff
@@ -1,3 +1,4 @@
 -r test.txt
 
 ipython
+ipdb
```

Comment:
This is useful to have in the dev setup for examining state in tests (although I am yet to work out how to live-debug servers)